### PR TITLE
Add notes for non-gsutil users.

### PR DESCRIPTION
**Pull request by duftler**

### CODE REVIEW

**Comment by duftler** [comment]
Closes #1211.

File: guides/operator/custom-boms.md [hunk 1]
```diff
@@ -170,6 +170,9 @@ As a result, a BOM at version `${VERSION}` can be found at
 subcomponent `${SUBCOMPONENT}` at version `${SUBCOMPONENT_VERSION}` can be
 found at `gs://${BUCKET}/${SUBCOMPONENT}/${SUBCOMPONENT_VERSION}/${PROFILE}`.
 
+If Halyard is unable to read from the GCS bucket, please see the troubleshooting
```
Comment (dorbin) [inline guides/operator/custom-boms.md]:
```suggestion
If Halyard can't read from the GCS bucket, please see the troubleshooting
```

File: guides/operator/custom-boms.md [hunk 2]
```diff
@@ -170,6 +170,9 @@ As a result, a BOM at version `${VERSION}` can be found at
...
+instructions [here](https://www.spinnaker.io/setup/quickstart/faq/#halyard-times-out-during-a-config-change).
```
Comment (dorbin) [inline guides/operator/custom-boms.md]: Instead of "...troubleshooting instructions [here]().", maybe just "...[troubleshooting instructions]()."

**Review by dorbin — APPROVED** [review]
Couple a suggestions, but lgtm anyway

**Pull request merged**